cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -74,6 +74,7 @@ public void export(Network network, Properties params, DataSource ds) {
         try {
             List<String> profiles = Parameter.readStringList(getFormat(), params, PROFILES_PARAMETER, defaultValueConfig);
             if (profiles.contains("EQ")) {
+                context.setExportEquipment(true);
```

Comment:
Shouldn't be used in `CgmesExport` for me but called in `EquipmentExport` for those directly using the classes